nan_json.h

```diff
@@ -87,7 +87,7 @@ class JSON {
     v8::Isolate* context_or_isolate = v8::Isolate::GetCurrent();
 #endif  // NODE_MODULE_VERSION > NODE_6_0_MODULE_VERSION
     result = v8::JSON::Parse(context_or_isolate, json_string);
-#endif  // NODE_MODULE_VERSION == NODE_0_12_MODULE_VERSION
+#endif  // NODE_MODULE_VERSION >= NODE_0_12_MODULE_VERSION && NODE_MODULE_VERSION <= IOJS_2_0_MODULE_VERSION
```

Comment:
Should this comment also be split into 2 lines?

Reply:
Depends on if the linter is smart enough to comprehend it. Try `make lint` and see what it says.
I am guessing it will complain about too long line or missing comment...